projects/packages/forms/declarations.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module '*.png';
+declare module '*.svg';
+declare module '*.jpeg';
+declare module '*.jpg';
+declare module '*.scss';
+declare module '*.css';
+declare module '*.mdx';
+declare module '*.svg';
```

Comment:
Would it make any sense to add the TS support in a separate PR? That one could include all the TS/build changes here, as well as some of the pure TS file renaming in this PR now that are a bit separate from updating the About page.